portality/events/consumer_utils.py

```diff
@@ -0,0 +1,23 @@
+from portality import models
+from portality.bll import exceptions
+from portality.core import app
+
+
+def parse_application(application_raw: dict) -> models.Application:
+    try:
+        return models.Application(**application_raw)
+    except Exception as e:
+        raise exceptions.NoSuchObjectException(
+            "Unable to construct Application from supplied source - data structure validation error, {x}".format(x=e)
+        )
+
+
+def parse_email_issns(issns: list) -> str:
+    issns = issns or []
+    return ", ".join(issn for issn in issns)
```

Comment:
If we really feel this should be a function in its own right, I would place this on the BibJSON object and call it something like "issns_as_text" or something.  That's `portality.models.v2.bibjson.JournalLikeBibJSON`

Reply:
we can move it to model layer for now.

But you will found that if some business requirements change, issns in email should show with other special format. Developer still need to create a `parse_email_issns` in `consumer_utils` because that is local logic of consumer module.